README.md

```diff
@@ -6,8 +6,9 @@
 [![Dependency Status](https://gemnasium.com/abonas/kubeclient.svg)](https://gemnasium.com/abonas/kubeclient)
 
 A Ruby client for Kubernetes REST api.
-The client supports GET, POST, PUT, DELETE on nodes, pods, secrets, services, replication controllers, namespaces, resource quotas, limit ranges, endpoints, persistent volumes, persistent volume claims, component statuses and service accounts.
+The client supports GET, POST, PUT, DELETE on all the entities available in kubernetes in both the core and group apis.
```

Comment:
https://kubernetes.io/docs/api/ doesn't talk of "entities", it says "resources" aka "objects".  OTOH "entities" terminology is all around Kubeclient, even baked into the  `client.all_entities` API...
 Maybe say "on all entities (nodes, pods, etc.)" just to be clear.

Consider also linking to https://kubernetes.io/docs/api-reference/v1/operations/ which actually lists the `/api` resources, and/or https://kubernetes.io/docs/reference/ which lists other `/apis` groups.

P.S. not necessarily now but would be nice to have a better example of group api use, not just constructing `/apis/batch` but also using it (`get_jobs` ?)